src/api/README.md

```diff
@@ -63,6 +63,7 @@ These API sets are:
 
 * `READ` - All query-related endpoints, they do not modify the state of the program
 * `STATUS` - A subset of `READ`, these endpoints report the application, network or blockchain status
+* `TXN` - Endpoints implementing transaction operations with side-effects that modify Skycoin node state
```

Comment:
should say:

Enables `/api/v1/injectTransaction` without enabling wallet endpoints